gradle.properties

```diff
@@ -1,6 +1,8 @@
 minimumXtextVersion = 2.9.0
 #We can't use 2.11 while we still want Java 6 support
 bootstrapXtextVersion = 2.9.1
-latestXtextVersion = 2.14.0
+latestXtextVersion = 2.25.0
+minimumGradleVersion = 4.3
+latestGradleVersion = 6.9.1
```

Comment:
i guess we can test with 7.2 too, cant we?

Reply:
That would require too many test changes right now, since the `compile` configuration and friends were removed.